src/foundry/common/documents/scene.mjs.d.ts

```diff
@@ -0,0 +1,372 @@
+      x: fields.NumberField<{ integer: true; nullable: true; initial: undefined }>;
+      y: fields.NumberField<{ integer: true; nullable: true; initial: undefined }>;
+      scale: fields.NumberField<{ nullable: true; min: 0.25; max: 3; initial: undefined }>;
+    }>;
+
+    /**
+     * The color of the canvas displayed behind the scene background
+     * @defaultValue `"#999999"`
+     */
+    backgroundColor: fields.ColorField<{ initial: "#999999" }>;
+
+    // Grid Configuration
+    /**
+     * Grid configuration for the scene
+     * @defaultValue `CONST.GRID_TYPES.SQUARE`
+     */
+    grid: fields.SchemaField<{
+      type: fields.NumberField<{
+        required: true;
+        choices: CONST.GRID_TYPES[];
+        initial: typeof CONST.GRID_TYPES.SQUARE;
+        validationError: "must be a value in CONST.GRID_TYPES";
+      }>;
+      size: fields.NumberField<{
+        required: true;
+        nullable: false;
+        integer: true;
+        min: typeof CONST.GRID_MIN_SIZE;
+        initial: 100;
+        validationError: `must be an integer number of pixels, ${typeof CONST.GRID_MIN_SIZE} or greater`;
+      }>;
+      color: fields.ColorField<{ required: true; nullable: false; initial: "#000000" }>;
+      alpha: fields.AlphaField<{ initial: 0.2 }>;
+      distance: fields.NumberField<{ required: true; nullable: false; positive: true; initial: () => number }>;
+      units: fields.StringField<{ initial: () => string }>;
+    }>;
+
+    // Vision and Lighting Configuration
+    /**
+     * Do Tokens require vision in order to see the Scene environment?
+     * @defaultValue `true`
+     */
+    tokenVision: fields.BooleanField<{ initial: true }>;
+
+    /**
+     * Is a global source of illumination present which provides dim light to all
+     * areas of the Scene?
+     * @defaultValue `true`
+     */
+    fogExploration: fields.BooleanField<{ initial: true }>;
+
+    /**
+     * The ambient darkness level in this Scene, where 0 represents midday
+     * (maximum illumination) and 1 represents midnight (maximum darkness)
+     * @defaultValue `Date.now`
+     */
+    fogReset: fields.NumberField<{ nullable: false; initial: number }>;
+
+    /**
+     * A darkness threshold between 0 and 1. When the Scene darkness level
+     * exceeds this threshold Global Illumination is automatically disabled
+     * @defaultValue `false`
+     */
+    globalLight: fields.BooleanField;
+
+    /**
+     * Should fog exploration progress be tracked for this Scene?
+     * @defaultValue `null`
+     */
+    globalLightThreshold: fields.AlphaField<{ nullable: true; initial: null }>;
+
+    /**
+     * The timestamp at which fog of war was last reset for this Scene.
+     * @defaultValue `0`
+     */
+    darkness: fields.AlphaField<{ initial: 0 }>;
+
+    /**
+     * A special overlay image or video texture which is used for fog of war
+     * @defaultValue `null`
+     */
+    fogOverlay: fields.FilePathField<{ categories: ["IMAGE", "VIDEO"] }>;
```

Comment:
As with the other PR, I'm not entire sure if this works or it should be `("IMAGE" | "VIDEO")[]` instead.

Reply:
I've gone ahead and set it to `("IMAGE" | "VIDEO")[]`. I'm honestly not sure what's going on with `FilePathField`. I'll have to dig into that more closely sometime.